### PR TITLE
BugFix for Issue #102 - Crash when turning on In-Flight trajectory display while in Map view

### PR DESCRIPTION
**Pull request by PiezPiedPy**
Used a work around for [Issue #102](https://github.com/neuoy/KSPTrajectories/issues/102) suggested by @fat-lobyte.
 Fix works by simply setting `line.enabled` to false and checking we are not in map view before changing any vertices and setting `line.enabled` to true.

Also fixed some syntax in MapGUI.cs

### CODE REVIEW

**Comment by fat-lobyte** [comment]
Looks good! I'll test and merge when I'm back from vacation :)

File: Plugin/MapGUI.cs [hunk 1]
```diff
@@ -162,12 +165,18 @@ private void FixedUpdate()
                     && lastPatchBody == targetBody && traj.targetPosition.HasValue)
                 {
                     // Get Latitude and Longitude from impact position
+                    double impactLat;
+                    double impatLon;
+                    double impactAlt;
                     impactPos = lastPatch.impactPosition.Value + lastPatchBody.position;
-                    lastPatchBody.GetLatLonAlt(impactPos, out double impactLat, out double impatLon, out double impactAlt);
```
Comment (fat-lobyte) [inline Plugin/MapGUI.cs]: This, I don't quite get. My Visual Studio suggested to inline the "out" variable declarations, and I think that does look better this way. Why back?

Comment (neuoy) [inline Plugin/MapGUI.cs]: If I'm not mistaken, this is a new syntax and won't work in previous versions of visual studio, that might be the reason.

Comment (fat-lobyte) [inline Plugin/MapGUI.cs]: Oh, ok. Sorry, that is my bad then.

**Comment by fat-lobyte** [comment]
Thanks for the PR, merged!

I'm thinking of putting out a new release rather soon (especially because of the precision fix). What do you think?

**Comment by Gordon-Dry** [comment]
I would appreciate that.

**Comment by PiezPiedPy** [comment]
>Thanks for the PR, merged!
I'm thinking of putting out a new release rather soon (especially because of the precision fix). What do you think?

 Can't see why not, the GUI is a while off yet so no need to wait round for that and peeps seem to be waiting for the bug fix so I'll vote for a release ;)

**Comment by fat-lobyte** [comment]
Okay, I'll get to it tomorrow (releasing late in the evening is not a great idea ;) )

> Can't see why not, the GUI is a while off yet so no need to wait round for that

By the way, I'm not sure how you code it, but if it's possible with minimal changes to make the users switch between the GUI versions, then I would like to ship both GUIs for 1-2 Releases to smooth out the transition and gather some feedback.

If you are tearing out the old GUI completely, that obviously doesn't make sense. Then we'll just... *surprise* the users ;)

**Comment by fat-lobyte** [comment]
The deed is done.

**Comment by PiezPiedPy** [comment]
>By the way, I'm not sure how you code it, but if it's possible with minimal changes to make the users switch between the GUI versions, then I would like to ship both GUIs for 1-2 Releases to smooth out the transition and gather some feedback.

I can do that without too much hassle, should be just a case of adding a button to the gui's

**Pull request merged**